cms/static/js/views/xblock_outline.js

```diff
@@ -44,6 +44,19 @@ define(["jquery", "underscore", "gettext", "js/views/baseview", "js/views/utils/
                 this.renderTemplate();
                 this.addButtonActions(this.$el);
                 this.addNameEditor();
+
+                if (this.model.get('is_header_visible') !== null) {
+                    // Currently an entrance exam does not contains subsection.
+                    // So we are setting the locator in order to show expanded.
```

Comment:
This comment, "Currently an entrance exam does not contains subsection." is confusing. It does contain a subsection, but the header controls are hidden, correct? Please update the comment.